### PR TITLE
Windows: use gsl 2.7 from rwinlib

### PR DESCRIPTION
**Pull request by jeroen**
Uses same libgsl as RcppGSL.

### CODE REVIEW

File: src/Makevars.win [hunk 1]
```diff
@@ -1,3 +1,10 @@
-PKG_CPPFLAGS = -I. -I../inst/include
-## Use the R_HOME indirection to support installations of multiple R version
-PKG_LIBS = $(shell "${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" -e "RcppGSL:::LdFlags()")
```
Comment (eddelbuettel) [inline src/Makevars.win:3]: Isn't using this programmatically and dynamically better?

If we test for "is it the same" should that not happen at configure time?

If it works it works ... but seems off.

Comment (jeroen) [reply src/Makevars.win:3]: I don't think you're really using this dynamically? `RcppGSL:::LdFlags()` just yields a string with `"-lgsl -lgslcblas"` but does not provide the library in any way. 

Given that you're linking against libgsl (and not R functions exported by RcppGSL) I think this PR is much easier.

Comment (eddelbuettel) [reply src/Makevars.win:3]: Maybe.  The code is old.  It should also supply and and all -L flags but might not know about them.  So yes, the fixed setup (for Windows where things are fixed) is better.

Ditto for RDieHarder. I can look into that in a day or two unless you get there sooner.

Comment (jeroen) [reply src/Makevars.win:3]: Yeah, I tested that this works both with the stable and ucrt toolchains.

**Review by eddelbuettel — APPROVED** [review]
Looks good, and I'll just make some minor whitespace edits as I did for RcppGSL.  So thanks for the PR!

**Pull request merged**